scene/3d/gi_probe.cpp

```diff
@@ -490,6 +492,14 @@ PoolVector<Face3> GIProbe::get_faces(uint32_t p_usage_flags) const {
 	return PoolVector<Face3>();
 }
 
+String GIProbe::get_configuration_warning() const {
+
+	if (OS::get_singleton()->get_current_video_driver() == OS::VIDEO_DRIVER_GLES2) {
+		return TTR("GIProbes are not supported by the GLES2 video driver.\nUse a Baked LightMap instead.");
```

Comment:
Would be better to use the exact node name, so `BakedLightmap`. Otherwise translators will make a huge mess of this warning. (I can already see people wonder what a "Carte de Lumière Cuite" would be... :))

Reply:
Good point. Done!